### PR TITLE
test for DAX encyption being enabled

### LINKED ISSUE #164: Add new check: DynamoDB Accelerator (DAX) encryption at rest enabled
https://www.terraform.io/docs/providers/aws/r/dax_cluster.html#server_side_encryption
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dax-cluster.html#cfn-dax-cluster-ssespecification

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Simple test for property being set within a block
Addresses test for DAX encryption being enabled: Partially Resolves #164 as only solves for Terraform not CF.

### CODE REVIEW

File: checkov/terraform/checks/resource/aws/DAXEncryption.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class DAXEncryption(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure DAX is encrypted at rest (default is unencrypted)"
+        id = "CKV_AWS_47"
+        supported_resources = ['aws_dax_cluster']
+        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```
Comment (schosterbarak) [inline checkov/terraform/checks/resource/aws/DAXEncryption.py]: `CheckCategories.ENCRYPTION` might be a better fit here

File: tests/terraform/checks/resource/aws/test_DynamodbRecovery.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from checkov.common.models.enums import CheckResult
 
 
-class TestEBSEncryption(unittest.TestCase):
+class TestDynamodbRecovery(unittest.TestCase):
```
Comment (schosterbarak) [inline tests/terraform/checks/resource/aws/test_DynamodbRecovery.py]: thanks :)

File: checkov/terraform/checks/resource/aws/DAXEncryption.py (hunk 1 above)
Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/aws/DAXEncryption.py]: ctrl+c ctrl+v .... fixed it.

**Comment by schosterbarak** [comment]
thank you! @JamesWoolfenden

**Pull request merged**